documentation/src/main/asciidoc/topics/ref_jgroups_ports.adoc

```diff
@@ -21,7 +22,3 @@
 |JGroups RELAY2 for cross-site replication
```

Comment:
RELAY2 doesn't have a default port. If not configured, it will use 7800++

Reply:
added some clarification. hopefully changes make sense. or we can just remove the cross-site bits.